frontend/packages/knative-plugin/src/hooks/index.ts

```diff
@@ -1,2 +1,37 @@
-export * from './useEventSourceModelsWithAccess';
-export * from './useEventSourceStatus';
+import { getExecutableCodeRef } from '@console/dynamic-plugin-sdk/src/coderefs/coderef-utils';
+
+export const eventSourceBreadcrumbsProvider = getExecutableCodeRef(() =>
```

Comment:
i don't think it's a good idea to replace index.ts with coderef hook exports here. This can be confusing as general imports would expect to work from `/hooks` directly which will not happen. Also this leads to changing older imports from the hooks folder which is not right. Better to create a new file or something that has all coderefs like these and then import from that.

Reply:
moved these to `src/providers`